docs/model/validation.rst

```diff
@@ -1,125 +1,121 @@
+* **Maximum emissions constraint**: Grid + PV + Diesel Generator (data: `set 1 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_maximum_emissions_None>`__, `set 2 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_maximum_emissions_low>`__, `set 3 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_maximum_emissions_low_grid_RE_100>`__/`pytest <https://github.com/rl-institut/multi-vector-simulator/blob/f459b35da6c46445e8294845604eb2b683e43680/tests/test_benchmark_constraints.py#L121>`__): Emissions are limited by constraint, more PV is installed to reduce emissions. For RE share of 100 % in grid, more electricity from the grid is used
 
-More tests can still be implemented with regard to:
+* **Parser** converting an energy system model from EPA to MVS (`data <https://github.com/rl-institut/multi-vector-simulator/blob/dev/tests/benchmark_test_inputs/epa_benchmark.json>`__/`pytest <https://github.com/rl-institut/multi-vector-simulator/blob/dev/tests/test_benchmark_scenarios.py>`__)
 
-* The investment model within the MVS
+* **Stratified thermal energy storage** (`data <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Feature_stratified_thermal_storage>`__/`pytest <https://github.com/rl-institut/multi-vector-simulator/blob/dev/tests/test_benchmark_stratified_thermal_storage.py>`__): With fixed thermal losses absolute and relative reduced storage capacity only if these losses apply
 
-* Components with two input sources
+* **Net zero energy (NZE) constraint**: Grid + PV and Grid + PV + Heat Pump (data `set 1 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_net_zero_energy_true>`__, `set 2 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_net_zero_energy_False>`__, `set 3 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_net_zero_energy_sector_coupled_true>`__, `set 4 <https://github.com/rl-institut/multi-vector-simulator/tree/dev/tests/benchmark_test_inputs/Constraint_net_zero_energy_sector_coupled_False>`__/`pytest <https://github.com/rl-institut/multi-vector-simulator/blob/dev/tests/test_benchmark_constraints.py>`__): Degree of NZE >= 1 when constraint is used and degree of NZE < 1 when constraint is not used.
 
-Sensitivity Analysis Tests
-##########################
+.. note::
+    Benchmark test inputs are available in the codebase folders within :code:`tests/benchmark_test_inputs`. They can also be used to get to know the MVS. The benchmark test assertions are provided as :code:`pytests` in a number of files in :code:`tests` with the naming convention :code:`test_benchmark_*`.
```

Comment:
```suggestion
    Benchmark test input data is available in the codebase folders within :code:`tests/benchmark_test_inputs`. It can also be used as simple example cases to get to know the MVS. The benchmark test assertions are provided as :code:`pytests` in a number of files in :code:`tests` with the naming convention :code:`test_benchmark_*`.
```